tumblr-photo-video-ripper.py

```diff
@@ -185,34 +190,76 @@ def _download_media(self, site, medium_type, start):
         if not os.path.isdir(target_folder):
             os.mkdir(target_folder)
 
-        base_url = "http://{0}.tumblr.com/api/read?type={1}&num={2}&start={3}"
+        # liked posts:
```

Comment:
Should move to two functions, such as `download_likes()` and `download_posts()`. And for each, there will be photos and videos.